tests/run-make/print-to-output/rmake.rs

```diff
@@ -0,0 +1,66 @@
+//! This checks the output of some `--print` options when
+//! output to a file (instead of stdout)
+
+extern crate run_make_support;
+
+use std::ffi::OsString;
+
+use run_make_support::{rustc, target, tmp_dir};
+
+fn main() {
+    // Printed from CodegenBackend trait impl in rustc_codegen_llvm/src/lib.rs
+    check(
+        /*target*/ &target(),
+        /*option*/ "relocation-models",
+        /*includes*/ &["dynamic-no-pic"],
+    );
```

Comment:
Can also become a struct

```rs
struct Args {
    target: &'static str,
    option: &'static str,
    includes: &'static [&'static str],
}
```

Reply:
And here I also changed it to use a struct but without function arg de-structuring. Let me know if you have a preference.